ci.py

```diff
@@ -449,7 +449,7 @@ def load_yml(yml_path):
         with open(yml_path) as compose_file:
             compose_dict = yaml.safe_load(compose_file)
             return compose_dict
-    except:
+    except(RuntimeError, IOError):
```

Comment:
Use `EnvironmentError` instead of `IOError`
https://docs.python.org/2/library/exceptions.html#exceptions.EnvironmentError
and catch also error from yaml